.github/workflows/release.yml

```
@@ -0,0 +1,16 @@
+name: Release
+on:
+  pull_request:
+    branches: [ $default-branch ]
+    types: closed
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && true == contains(join(github.event.pull_request.labels.*.name), 'Release')
+    steps:
+    - uses: actions/checkout@v4
+    - uses: elementary/actions/release@main
+      env:
+        GIT_USER_TOKEN: "${{ secrets.GIT_USER_TOKEN }}"
+        GIT_USER_NAME: "elementaryBot"
+        GIT_USER_EMAIL: "builds@elementary.io"
```

This makes sure we push the release to a separate stable branch which the LP recipe will build from

```suggestion
        GIT_USER_EMAIL: "builds@elementary.io"
       with:
         release_branch: 'noble'
```

---

Done